src/ripple/app/ledger/impl/LedgerToJson.cpp

```diff
@@ -65,6 +65,10 @@ void fillJson(Object& json, LedgerInfo const& info, bool bFull)
     json[jss::hash] = to_string (info.hash);
     json[jss::totalCoins] = to_string (info.drops);
     json[jss::accepted] = ! info.open;
+    json[jss::close_flags] = info.closeFlags;
+
+    if (info.parentCloseTime)
```

Comment:
err..that's not quite right. Consensus sometimes does not come to a conclusion about the close time. For these cases the close time is set to zero. Meaning "no consensus reached."